frontend/source/components/SignUpComponent/SignUpComponent.css

```diff
@@ -0,0 +1,176 @@
+.sign-up-container {
+    background-color: #EAF6FF;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 100vw;
+    height: 100vh;
+    font-size: large;
+}
+
+.sign-up-container>h2 {
+    color: #4A90E2;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-size: xx-large;
+    margin-bottom: 5px;
+
+}
+
+
+.sign-up {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* justify-content: space-around; */
+    background-color: #FFFFFF;
+    border: #DDDDDD 1px solid;
+    padding: 20px;
+    width: 30%;
+    height: 60%;
+    margin-bottom: 20px;
+}
+
+.sign-up>h3 {
+    font-size: x-large;
+    text-align: center;
+    max-width: 70%;
+
+}
+
+input {
+    margin-top: 4px;
+    padding: 5px;
+}
+
+.input-block {
+    display: flex;
+    margin-bottom: 3%;
+    margin-top: 3%;
+    flex-direction: column;
+    justify-content: space-evenly;
+    padding: 20px;
+    border: #DDDDDD 1px solid;
+    background-color: #f9f9f9;
+    height: 90%;
+    width: 60%;
+}
+
+.inner_input {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+}
+
+/* .inner_input.toggle {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+} */
+
+/* .toggle>label {
+    margin-top: 1em;
+} */
+
+.inner_input>input {
+    padding: 3px;
+    padding-left: 10px;
+}
+
+#noti_pref {
+    height: 2.5em;
+    width: 2.5em;
+    margin: 0;
+    accent-color: #2ECC71;
+
+}
+
+
+.submit-button {
+    background-color: #2ECC71;
+    padding: 1% 3% 1% 3%;
+    border-radius: 20px;
+    color: #FFFFFF;
+    font-size: large;
+    border: none;
+    margin-top: 7px;
+}
+
+.submit-button:focus {
+
+    background-color: #239452;
+}
+
+.login_route {
+    margin-top: 0;
+    padding-top: 2px;
+}
+
+.login_route_btn {
+    background-color: inherit;
+    border: none;
+    text-decoration: underline;
+    padding: 0;
+    font-size: large;
+}
```

Comment:
Consider adding "cursor: pointer;" to the button. It'll improve clarity for user.